static/js/publisher/market/markdown.js

```diff
@@ -0,0 +1,13 @@
+export default function() {
+  const markdownLink = document.querySelector(".js-toggle-markdown");
+  const markdownInfo = document.querySelector(".js-markdown");
+
+  markdownLink.addEventListener("click", function(e) {
+    e.preventDefault();
+    if (markdownInfo.classList.contains("u-hide")) {
```

Comment:
This is no longer necessary as `classList` has a `toggle` method.

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList